commit ec17ee7e24b2
Author: Kaku Kuo <kaku@mozilla.com>
Date:   Thu Sep 21 17:47:45 2017 +0800

    Bug 1401909 - Turn on preference of ShutdownDecoder feature on desktop version release and beta channel. r=bwu, a=sledru
    
    MozReview-Commit-ID: JB3RmyjhvK9
    
    --HG--
    extra : source : 27953b6cbc9b9bff5fb14cb0e69b492f08e62da8
---
 modules/libpref/init/all.js | 4 ----
 1 file changed, 4 deletions(-)

diff --git modules/libpref/init/all.js modules/libpref/init/all.js
index 6d66d27d987c..c74659f8d4a5 100644
--- modules/libpref/init/all.js
+++ modules/libpref/init/all.js
@@ -432,11 +432,7 @@ pref("media.decoder-doctor.wmf-disabled-is-failure", false);
 pref("media.decoder-doctor.new-issue-endpoint", "https://webcompat.com/issues/new");
 
 // Whether to suspend decoding of videos in background tabs.
-#ifdef RELEASE_OR_BETA
-pref("media.suspend-bkgnd-video.enabled", false);
-#else
 pref("media.suspend-bkgnd-video.enabled", true);
-#endif
 // Delay, in ms, from time window goes to background to suspending
 // video decoders. Defaults to 10 seconds.
 pref("media.suspend-bkgnd-video.delay-ms", 10000);
